src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -1,5 +1,11 @@
 package org.zalando.nakadi.domain;
 
 public enum EventPublishingStep {
-    NONE, VALIDATING, ENRICHING, PARTITIONING, PUBLISHING
+    NONE,
+    VALIDATING,
+    ENRICHING,
+    PARTITIONING,
+    PUBLISHING,
+    PUBLISHED,
+    FAILED_PUBLISH
```

Comment:
We can't change this. This can break a lot of clients which expect only the specific values. This will be backward-incompatible change.
https://github.com/zalando/nakadi/blob/master/api/nakadi-event-bus-api.yaml#L1597